samples/ios/app/glean-sample-app/Info.plist

```diff
@@ -41,5 +41,12 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSAppTransportSecurity</key>
```

Comment:
nit: can we add a comment here mentioning why these permissions are required? (i.e. sending pings to a local server on UI tests)

Reply:
This file is edited through the GUI. I'm not sure how nice it plays with adding comments :/

Reply:
Oh, eugh. Ok! No problem then!

Reply:
Just tested it. Xcode will removed the comment if someone opens and saves the Info.plist inside Xcode.